client/src/api/projects.js

```diff
@@ -0,0 +1,30 @@
+import httpClient from './httpClient';
+
+export const addProject = async (id, projectData) => {
+    const url = `/api/projects/add/${id}`;
+    const token = localStorage.getItem('jwtToken')
+    const authOptions = {
```

Comment:
The interceptor in the api should theoretically be modifying the header to contain the token if it exists in local storage.

Reply:
The issue is that at times the token does not load consistently. It may be an issue with my machine but there would be times that it gave the token, but other times that it didn't. A jwtToken is consistently added to localStorage, but in the httpClient folder, there are times where it logged as null for me despite it showing up in localStorage.

Reply:
strange. Thanks for letting me know. I'll create an issue for it.

Reply:
@DevDelvo I know localstorage.getItem() returns null if the key is not found in localstorage, but 🤔